static/app/views/metrics/metricQueryContextMenu.tsx

```diff
@@ -139,18 +139,22 @@ export function MetricQueryContextMenu({
         leadingItems: [<IconSettings key="icon" />],
         key: 'settings',
         label: t('Metric Settings'),
-        disabled: !isCustomMetric({mri: metricsQuery.mri}),
         onAction: () => {
           trackAnalytics('ddm.widget.settings', {
             organization,
           });
           Sentry.metrics.increment('ddm.widget.settings');
-          navigateTo(
-            `/settings/projects/:projectId/metrics/${encodeURIComponent(
-              metricsQuery.mri
-            )}`,
-            router
-          );
+          if (isCustomMetric({mri: metricsQuery.mri})) {
```

Comment:
FYI: This will also be true for span-based metrics.
But no worries, will handle this in my PR.